board/drivers/can.h

```diff
@@ -130,29 +130,24 @@ void can_set_speed(uint8_t can_number) {
   CAN_TypeDef *CAN = CANIF_FROM_CAN_NUM(can_number);
   uint8_t bus_number = BUS_NUM_FROM_CAN_NUM(can_number);
 
-  while (true) {
```

Comment:
does not seem like the while was needed.

Reply:
yea, that's fine. can init may fail, but looping is probably the wrong choice anyway